src/tox.ini

```diff
@@ -314,7 +314,7 @@ passenv = RUFF_OUTPUT_FORMAT
 #
 commands =
     ruff check --ignore E402,E721,E731,E741,E742,E743,F401,F402,F403,F405,F821,F841,I001,PLC0206,PLC0208,PLC2401,PLC3002,PLE0302,PLR0124,PLR0402,PLR0911,PLR0912,PLR0913,PLR0915,PLR1704,PLR1711,PLR1714,PLR1716,PLR1736,PLR2004,PLR5501,PLW0120,PLW0211,PLW0602,PLW0603,PLW0642,PLW1508,PLW1510,PLW2901,PLW3301 {posargs:{toxinidir}/sage/}
-    ruff check --preview --select E111,E115,E21,E221,E222,E225,E227,E228,E25,E271,E272,E275,E302,E303,E305,E306,E401,E502,E701,E702,E703,E71,W291,W293,W391,W605 {posargs:{toxinidir}/sage/}
+    ruff check --preview --select E111,E115,E21,E221,E222,E225,E227,E228,E25,E271,E272,E275,E302,E303,E305,E306,E401,E502,E701,E702,E703,E71,W291,W293,W391,W605,TC {posargs:{toxinidir}/sage/}
```

Comment:
This seems to have no effect on the CI run. It would be nice if there was one place where the linter rules we use are listed rather than duplicating it in `src/tox.ini` and `.github/workflows/lint.yml`. Not a big deal for this PR, just mentioning it for things to think about for future work.